### PR TITLE
Transient contact condition

### PR DESCRIPTION
**Pull request by IvarStefansson**
Two main changes:
   Initial state subtracted for the tangential displacement jumps in the contact discretization
   Fixed sign mistake in DivU

### CODE REVIEW

File: src/porepy/models/contact_mechanics_model.py [hunk 1]
```diff
@@ -430,15 +430,13 @@ def newton_iteration(assembler, setup, u0, tol=1e-14, solver=None):
     A, b = assembler.assemble_matrix_rhs()
     print("max A: {0:.2e}".format(np.max(np.abs(A))))
     print(
-        "max: {0:.2e} and min: {1:.2e} A sum: ".format(
+        "max: {0:.2e} and min: {1:.2e} A sum.".format(
```
Comment (rbe051) [inline src/porepy/models/contact_mechanics_model.py]: I think you can remove these print statements, or maybe better change then to  logger.debug(".......")

File: src/porepy/numerics/contact_mechanics/contact_conditions.py [hunk 2]
```diff
@@ -116,21 +120,39 @@ def discretize(self, g_h, g_l, data_h, data_l, data_edge):
...
         # two sides (which is really a difference since one of the sides have
         # its sign switched).
-        displacement_jump_global_coord = (
+        # The tangential displacements are relative to the initial state, which in the
+        # transient case equals the previous time step.
+        previous_displacement_iterate = data_edge[pp.STATE]["previous_iterate"][
+            self.mortar_displacement_variable
+        ]
+        previous_displacement_time = data_edge[pp.STATE][
+            self.mortar_displacement_variable
```
Comment (rbe051) [inline src/porepy/numerics/contact_mechanics/contact_conditions.py]: is self.mortar_displacement_keyword a better name?

Comment (IvarStefansson) [reply src/porepy/numerics/contact_mechanics/contact_conditions.py]: I don't think so, seeing as this is the stored variable.

File: src/porepy/models/contact_mechanics_model.py (hunk 1 above)
Comment (IvarStefansson) [reply src/porepy/models/contact_mechanics_model.py]: Good idea!

**Pull request merged**